### PR TITLE
Add samples for the "Right-size your GKE workloads at scale" tutorial

### PR DESCRIPTION
**Pull request by aburhan**
This PR adds samples for an upcoming tutorial, "Right-size your GKE workloads at scale".

### CODE REVIEW

**Review by bourgeoisor — CHANGES_REQUESTED** [review]
Could you make sure that all the files in this PR are related to the VPA tutorial in one way or another? I flagged a few irrelevant files but there may be some more.

As for the Dockerfile (for the metrics exporter), you'l need 3 things:
- A cloudbuild file (follow this example: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/security/wi-secrets/cloudbuild.yaml)
- The cloudbuild file added to the TF config (follow this example: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/terraform/google-cloud-build-triggers.tf#L194-L207)
- A workflows that builds the container (follow the example here: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/security-ci.yml)

File: gke-vpa-recommendations/scripts/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
```
[the remark this answers is not in the document]
Comment (aburhan) [reply gke-vpa-recommendations/scripts/Dockerfile:1]: Yes, this is for the hpa metrics exporter

File: gke-vpa-recommendations/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+scripts/k8s/metrics-exporter.yaml
```
[the remark this answers is not in the document]
Comment (aburhan) [reply gke-vpa-recommendations/.gitignore]: removed

File: gke-vpa-recommendations/LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+
```
[the remark this answers is not in the document]
Comment (aburhan) [reply gke-vpa-recommendations/LICENSE]: removed

File: gke-vpa-recommendations/scripts/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```
[the remark this answers is not in the document]
Comment (aburhan) [reply gke-vpa-recommendations/scripts/CONTRIBUTING.md]: removed

File: gke-vpa-recommendations/scripts/LICENSE [hunk 5]
```diff
@@ -0,0 +1,202 @@
+
```
[the remark this answers is not in the document]
Comment (aburhan) [reply gke-vpa-recommendations/scripts/LICENSE]: removed

**Comment by aburhan** [comment]
added ci workflow files and removed unnecessary files

**Review by bourgeoisor — APPROVED** [review]

**Pull request merged**